File: Nth fibonacci/O(n)_time_O(n)_space.py
def getNthFib(n,mem={1:0,2:1}):
    # Write your code here.
    if n in mem:
        return mem[n]
    else:
        mem[n]= getNthFib(n-1,mem)+getNthFib(n-2,mem)
        return mem[n]

File: Nth fibonacci/O(n)_time_O(n)_space.js
function getNthFib(n) {
  // Write your code here.
  let fib = [0, 1];
  for (let i = 2; i <= n; i++) {
    fib.push(fib[i - 1] + fib[i - 2]);
  }
  console.log(fib);
  return fib[n - 1];
}
